### PR TITLE
Add feature to configure timeouts on HTTP client

### PR DESCRIPTION
**Pull request by naushniki**

### CODE REVIEW

File: doc/integration/rest-assured/rest.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ rest { # there you describe configuration for default rest context
 
   # base url which will be prefixed for every request
   base-url = "http://myhost:port/path"
+  socketTimeout = 30000 #Socket timeout in milliseconds (optional)
```
Comment (saksmt) [inline doc/integration/rest-assured/rest.md]: Please use `cebab-case` in configs

Comment (saksmt) [reply doc/integration/rest-assured/rest.md]: Also there is support for duration in hocon

File: gradle.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
 group = run.smt.ktest
-version = 1.1.0
+version = 1.1.1
```
Comment (saksmt) [inline gradle.properties:2]: If you want this fix to be released quickly as bugfix you need to create PR in appropriate versioned develop, not just to develop. Also version should be changed in release, not in PR

**Review by saksmt — CHANGES_REQUESTED** [review]

File: doc/integration/rest-assured/rest.md [hunk 3]
```diff
@@ -28,6 +28,8 @@ rest { # there you describe configuration for default rest context
 
   # base url which will be prefixed for every request
   base-url = "http://myhost:port/path"
+  socket-timeout = 30000 #Socket timeout in milliseconds (optional)
```
Comment (saksmt) [inline doc/integration/rest-assured/rest.md:31]: You can use
```kotlin
config.getDuration("socket-timeout", TimeUnit.MILLISECONDS)
```
to obtain value in millis from human-readable config value (i.e. `10 seconds`)

Comment (naushniki) [reply doc/integration/rest-assured/rest.md:31]: Unfortunatelly, getDuration returns a Long value, but Int is needed.

**Pull request closed without merging**